src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx

```diff
@@ -56,3 +57,29 @@ export const EmptyStateWithBoxAiDisabled: StoryObj<typeof MetadataSidebarRedesig
         },
     },
 };
+
+export const MetadataInstanceEditorWithDefinedTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        await canvas.findByRole('heading', { name: 'Metadata' });
```

Comment:
I dont think we need to `findByRole` for the heading since we arent interacting with it and this is not a test

Reply:
true, deleted but had to add timeout for the button to make sure stories catch it when it renders